students/Wieslaw_Pucilowski/lesson02/grid_printer.py

```diff
@@ -0,0 +1,88 @@
+# Lesson02 assignment
+__author__ = "Wieslaw Pucilowski"
+
+# symbol global variable definition
+v_slash="|"
+cross="+"
+dash="-"
+space=" "
+
+# first task - print grid
+def print_h ():
+	print(cross,(dash+space)*4,cross, (dash+space)*4,cross)
+
+def print_m_h():
+	print(v_slash,(space+space)*4,v_slash, (space+space)*4,v_slash)
+
+def print_4f(f):
+	for i in range(4):
+		f()
+
+def print_gr():
+	print_h ()
+	print_4f(print_m_h)
+	print_h ()
+	print_4f(print_m_h)
+	print_h ()
+
+# second task - print_grid(n)
+
+def print_h2(n):
+	print(cross,(space+dash)*n,space+cross,(space+dash)*n,space+cross)
+
+def print_m_h2(n):
+	print(v_slash,(space+space)*n,space+v_slash, (space+space)*n,space+v_slash)
+
+def print_nf(f,n):
+	for i in range(n):
+		f(n)
+
+def print_grid(n):
+	m = n // 2
+	print_h2(m)
+	print_nf(print_m_h2,m)
+	print_h2(m)
+	print_nf(print_m_h2,m)
+	print_h2(m)
```

Comment:
Not quite - only getting a single line printed out.

Reply:
Recommend adding some notes to make it clear what the expected output should be.